archives/header.html

```diff
@@ -27,7 +27,7 @@ <h2>Sponsor and contributions</h2>
     <li>Camptocamp (main contributors)</li>
     <li>Andrea Aime (GeoServer contributions)</li>
     <li>QGIS (html summary with graph)</li>
-    <li>3Liz (QGIS trust contribution)</li>
+    <li>3Liz (QGIS trust contribution - removed)</li>
```

Comment:
why not completely removing the line (if it points to nothing afterwards) ? we should find it back in the git history anyway

Reply:
good question: I think this is good to keep memory of past contribution.

Reply:
you have it in the git history already ;-)

Reply:
well, I meant a marketing point of view, header.html is a public page

Reply:
fair enough, then